interface-definitions/interfaces-tinc.xml.in

```diff
@@ -0,0 +1,476 @@
+            <properties>
+              <help>ReplayWindow(Bytes,default:16)</help>
+            </properties>
+            <defaultValue>16</defaultValue>
+          </leafNode>
+          <leafNode name="strict-subnets">
+            <properties>
+              <help>When this option is enabled tinc will only use Subnet statements which are present in the host config files in the local /etc/tinc/netname/hosts/ directory. Subnets learned via connections to other nodes and which are not present in the local host config files are ignored</help>
+              <valueless/>
+            </properties>
+          </leafNode>
+          <leafNode name="tunnel-server">
+            <properties>
+              <help>When this option is enabled tinc will no longer forward information between other tinc daemons, and will only allow connections with nodes for which host config files are present in the local /etc/tinc/netname/hosts/ directory, Setting this options also implicitly sets StrictSubnets</help>
+              <valueless/>
+            </properties>
+          </leafNode>
+          <leafNode name="udp-rcv-buf">
+            <properties>
+              <help>Sets the socket receive buffer size for the UDP socket, in bytes. If unset, the default buffer size will be used by the operating system(Bytes)</help>
+            </properties>
+          </leafNode>
+          <leafNode name="udp-snd-buf">
+            <properties>
+              <help>Sets the socket send buffer size for the UDP socket, in bytes. If unset, the default buffer size will be used by the operating system(Bytes)</help>
+            </properties>
+          </leafNode>
+          <leafNode name="clamp-mss">
+            <properties>
+              <help>Enable Clamp MSS</help>
+              <valueless/>
+            </properties>
+          </leafNode>
+          <leafNode name="indirect-data">
+            <properties>
+              <help>The tinc daemons other than those specified by ConnectTo can directly establish a connection with you</help>
+              <valueless/>
+            </properties>
+          </leafNode>
+          <leafNode name="disable-PMTU-Discovery">
+            <properties>
+              <help>Disable PMTU Discovery Option</help>
+              <valueless/>
+            </properties>
+          </leafNode>
+          <leafNode name="TCP-Only">
+            <properties>
+              <help>Only use TCP connection</help>
+              <valueless/>
+            </properties>
+          </leafNode>
+          <leafNode name="compression-level">
+            <properties>
+              <help>Compression Level Option(default:9)</help>
+            </properties>
+            <defaultValue>9</defaultValue>
+          </leafNode>
+          <leafNode name="mac-length">
+            <properties>
+              <help>MAC Length(Bytes,default:4))</help>
+            </properties>
+            <defaultValue>4</defaultValue>
+          </leafNode>
```

Comment:
Where does this option come from?

Reply:
Host configuration list from tinc, but it seems to be defined in tinc.conf

Reply:
Maybe I can establish a soft connection in the hosts directory